draft-ietf-tls-tls13.md

```diff
@@ -2434,10 +2524,13 @@ identifier
 {: br}
 
 When the client offers a PSK cipher suite, it MUST also supply a
-PreSharedKeyExtension to indicate the PSK(s) to be used. If no
-such extension is present, the server MUST NOT negotiate
-a PSK cipher suite. If no suitable identity is present, the server
-MUST NOT negotiate a PSK cipher suite.
+PreSharedKeyExtension to indicate the pre-shared key(s) to be used.
+Servers negotiating TLS 1.3 for a ClientHello without this extension yet
+offering any PSK cipher suite MUST send a "missing_extension"
+alert message and close the connection.
+If no suitable identity is present, the server MUST NOT negotiate
+a PSK cipher suite and MAY respond with an "unknown_psk_identity"
+alert message.
```

Comment:
Didn't TLS1.2 allow for masking if the given identity is correct or not? I think many people may want that, even if it simply can't be made timing resistant...